### ISSUE TITLE
What should autocomplete do when no results are found?

### ISSUE THREAD

**Issue by Luke1982** [Luke1982/IDLinesUX#8]
Should give some feedback to the user as well.

**Comment by joebordes** [comment]
LDS says:

It’s recommended to provide a message when a list is empty. For example, “No items to display.” "No items found"

I'm not sure that is the current behaviour of our autocomplete functionality

**Comment by Luke1982** [comment]
I agree. Just need to think about how to implement this, so we can use translations. Right now there is no message. The autocomplete doesn't filter, since I think that should be MySQL/PHP's job. The autocomplete should just display what it's given.
